src/views/Gallery.vue

```diff
@@ -1,131 +1,144 @@
+      show-cancel-button="focus"
+      :placeholder="$t('gallery.search')"
+      inputmode="search"
+      enterkeyhint="search"
+      :value="searchValue"
+      @ion-change="searchValue = $event.target.value"
     />
     <ion-content scroll-x="false" scroll-y="false">
-     
-        <ion-grid>
-            <ion-row>
-                <ion-col size="3" v-for="(item, index) in filterPicture" :key="index"> 
-                  <ion-thumbnail :key="index">
-                    <img :src="item.file?.url" @click="viewPicture(item,index)"/>
-                  </ion-thumbnail> 
-                </ion-col>
-            </ion-row>
-        </ion-grid>
-    </ion-content>
-    <ion-content>
+      <ion-grid>
+        <ion-row>
+          <ion-col
+            size="4" size-lg="2" size-md="3"
+            v-for="(item, index) in filterPicture"
+            :key="index"
+            class="custom-card"
+          >
+            <div :key="index" class="grid-div">
+              <img
+                class="grid-img"
+                :src="item.file?.url"
+                @click="viewPicture(item, index)"
+              />
+            </div>
+          </ion-col>
+        </ion-row>
+      </ion-grid>
     </ion-content>
+    <ion-content> </ion-content>
   </ion-page>
 </template>
-
 <script lang="ts">
 import {
-  IonContent, IonPage,modalController,IonSearchbar,
-  IonCol,IonRow,IonGrid,IonThumbnail
-} from '@ionic/vue';
-import { defineComponent, computed } from 'vue';
-import { useStore } from '../stores/';
-import PictureView from '../components/picture-view.vue';
-import { Model } from '@/utils/model';
-// import PictureViewModal from '../components/picture-view-modal.vue';
-import TeamFilterHeader from '../components/team-filter-header.vue';
-
+  IonContent,
+  IonPage,
+  modalController,
+  IonSearchbar,
+  IonCol,
+  IonRow,
+  IonGrid,
+} from "@ionic/vue";
+import { defineComponent, computed } from "vue";
+import { useStore } from "../stores/";
+import PictureView from "../components/picture-view.vue";
+import TeamFilterHeader from "../components/team-filter-header.vue";
 export default defineComponent({
-  name: 'Gallery',
-  data(){
-      const pictureArray: any[] = [];     
-      return{
-          searchValue:'',
-          selectedTeam:0,
-          pictureArray,
-          teamValue:''
-      }
+  name: "Gallery",
+  data() {
+    const pictureArray: any[] = [];
+    return {
+      searchValue: "",
+      selectedTeam: 0,
+      pictureArray,
+      teamValue: "",
+    };
   },
-   setup() {
+  setup() {
     const store = useStore();
-    const pictureArr: any[]=[];
-    // const elem = store.getters["news/latest"];
+    const pictureArr: any[] = [];
     return {
-      elem: computed(() => store.getters["news/latest"]),
+      elem: computed(() => store.getters["feed/latestPosts"]),
+      temppictureArray: computed(() => store.getters["feed/latestPosts"]),
       store,
-      pictureArr
-    }
+      pictureArr,
+    };
```

Comment:
out of scope of this PR, but that's not how this should work. Follow up at https://github.com/TeamFranka/affinity/issues/208

Reply:
this endpoint shouldn't be used here, right? only "news/latest" should be used.

Reply:
"news/latest" doesn't have files to use.

Reply:
both are wrong, but that's what the other issue is about. keep it as is.

Reply:
can you send new endpoint?

Reply:
As said, leave it as is, we will do that later.

Reply:
since #215 has been merged now, the facilities are there and it is being loaded correctly. Please use the  `gallery/entries` getter and the `gallery/refresh` action to fetch on mount.